sys/net/application_layer/asymcute/asymcute.c

```diff
@@ -256,6 +256,8 @@ static void _on_req_timeout(void *arg)
         mutex_unlock(&req->lock);
         mutex_unlock(&con->lock);
         con->user_cb(req, ret);
+        /* clear request for further use */
+        req->con = NULL;
```

Comment:
Mh... the request is indeed already cleared in [`_req_remove()`](https://github.com/RIOT-OS/RIOT/blob/ad20c4f1ccf68f7535366854d7482cff2df3f36e/sys/net/application_layer/asymcute/asymcute.c#L151) above... I don't know, what I was thinking when doing this PR.